src/components/HomePage.js

```diff
@@ -14,9 +15,11 @@ const HomePage = () => {
 
   return (
     <div>
-      <h1>Home Page</h1>
-      <BookList books={books} />
-      <BookForm />
+      <Navigation />
+      <div className="content">
+        <BookList books={books} />
+        <BookForm />
```

Comment:
- [ ] Kindly make sure to use the same color as the Zepline design for the `Add Book` button so it looks identical to the provided design